src/EditorFeatures/CSharpTest/BlockCommentEditing/BlockCommentEditingTests.cs

```diff
@@ -342,7 +342,7 @@ public void InsertOnMiddleLine6()
             var expected = @"
     /*
   
-     * $$*
+     $$*
      */
```

Comment:
this behavior is strictly better.  none of the unnecessary added `*`s.  we just align the existing `*` with the comment.